pkg/model/config/config_test.go

```diff
@@ -147,58 +192,90 @@ var _ = Describe("PluginConfig", func() {
 		Expect(config.DecodePluginConfig(key, &pluginConfig)).To(Succeed())
 		Expect(pluginConfig).To(Equal(expectedPluginConfig))
 	})
+
+	It("should Marshal and umarchal a plugin", func() {
```

Comment:
Are these cases testing anything added by this PR?

Reply:
Moved to another PR: https://github.com/kubernetes-sigs/kubebuilder/pull/1844/files